interpreter.cc

```diff
@@ -545,6 +545,14 @@ Obj ClosureDecl::interpretMethod(Interpreter::Context &c, Method *mth, Obj self,
 	{
 		c.setSymbol(*param.get(), &args[i++]);
 	}
+	i = 0;
+	Obj *locals = new Obj[decls.size()];
```

Comment:
This is created with new and so isn't visible to the garbage collector.  You need to allocate scanned memory instead.  Also, please consider using `unique_ptr` instead of manual deallocation.  `new` and `delete` should almost never appear in modern C++ code outside of standard library headers.